database/seeds/permissions.seed.json

```diff
@@ -27,18 +27,21 @@
     "user_details-own:delete",
     "user_details-own:read",
     "user_details-own:write",
+    "user_details-sibling:read",
+    "user_details-sibling:write",
     "visit_activities-child:read",
     "visit_activities-own:delete",
     "visit_activities-own:read",
     "visit_activities-own:write",
     "visit_logs-child:read",
-    "visit_logs-own:read",
-    "visit_logs-own:write",
+    "visit_logs-child:write",
     "volunteer_logs-child:read",
```

Comment:
Don't we also need `write` and `delete` versions of `volunteer_logs-child`?

Reply:
Yup, I forgot org_admin are allowed to login to the volunteer app.